### PR TITLE
Use promoted tripleo, add octavia support.

### PR DESCRIPTION
**Pull request by imain**
Also more lolcat :)  and set hostname properly to avoid nova placement erroring out.

### CODE REVIEW

**Comment by imain** [comment]
Also I had a successful deployment with this set up on a freshly provisioned dsal machine.

File: 01_install_requirements.sh [hunk 1]
```diff
@@ -21,20 +21,25 @@ if [ ! -f openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz ]; th
...
-cd
-#sudo tripleo-repos current
-#sudo tripleo-repos current-tripleo
+popd
+
+# Trying to use current-tripleo, which is the one that gets
+# promoted through CI (more stable).  Sometimes we need to switch
+# to current-tripleo-dev which is closer to master.
+#sudo tripleo-repos current-tripleo-dev
+sudo tripleo-repos current-tripleo
```
Comment (flaper87) [inline 01_install_requirements.sh]: I like this! thanks

File: 01_install_requirements.sh [hunk 2]
```diff
@@ -43,3 +48,13 @@ fi
 cd $SCRIPTDIR
 cp -rv /usr/share/openstack-tripleo-heat-templates ./tripleo-heat-templates
 
+# Patch to make octavia work in standalone:
+# https://review.openstack.org/#/c/628957/6
+pushd tripleo-heat-templates
+cat ../202ca5a_octavia_patch.diff | patch -p1
```
Comment (flaper87) [inline 01_install_requirements.sh]: What do you think about running `git init` in tht and then apply the patch? @hardys proposed something like this in one of my previous PRs as it's common to have to modify tht and save changes.

Comment (imain) [reply 01_install_requirements.sh]: I think it's a good idea if it works :).  I'd consider just merging this and changing it over after tho.  Unless you want to update it.. I'm done for today :).

Comment (flaper87) [reply 01_install_requirements.sh]: We can wait until tomorrow :smile: 

What I like of the `git init` workflow is that we won't have to carry `.patch` files around in ocp-doit.

**Pull request merged**